casper/src/test/scala/coop/rchain/casper/ValidateTest.scala

```diff
@@ -415,4 +415,26 @@ class ValidateTest
 
     activeRuntime.close()
   }
+
+  "Field format validation" should "suceed on a valid block and fail on empty fields" in {
```

Comment:
succeed
What do you use as your IDE? Is there a spellcheck function?

Reply:
Oops. I mostly use idea and it indeed has a spellcheck function. It's just me being blind.